code/controllers/_rotatable.py

```diff
@@ -19,3 +19,6 @@ def center(self) -> Point:
 
     def intersect(self, other: 'RotatableController') -> bool:
         return self.RADIUS + other.RADIUS > get_distance_between_points(self.center, other.center)
+
+    def get_distance(self) -> int:
+        return self._distance
```

Comment:
тут лучше `@property` использовать
```suggestion
    @property
    def distance(self) -> int:
        return self._distance
```